apps/explorer/src/components/OwnedObjects/ListView.tsx

```diff
@@ -54,21 +58,23 @@ function ListViewItemContainer({ obj }: { obj: SuiObjectResponse }) {
 
 	return (
 		<ListViewItem
-			asset={
+			objectId={objectId!}
+			assetCell={
 				<>
 					<ObjectVideoImage
+						disabled
 						title={name}
 						subtitle={type}
 						src={displayMeta?.image_url || ''}
 						video={video}
 						variant="xs"
-						open={open}
-						setOpen={setOpen}
+						open={false}
+						setOpen={() => {}}
```

Comment:
did you intend to leave this empty function?

Reply:
also curious what this change is for

Reply:
Yea this is intended. I just want the component to render the `image` or `video` without using the preview function